.travis.yml

```diff
@@ -2,3 +2,8 @@ language: node_js
 node_js: node
 script:
 - npm run test:lint
+- 'if [ "$TRAVIS_BRANCH" == "master" ]; then frauci-update-version; fi'
```

Comment:
Won't this just always update the minor version? IMO it would be ideal if we could somehow keep the same workflow we've been using of using tagged releases in the GitHub UI, and somehow getting that value into the `package.json`...

Reply:
I believe you can use the notation [increment major] or [increment patch] in your merge message to determine what part of the version is bumped.

https://github.com/Brightspace/frau-ci#overview

Hmm, I'm not sure about hooking into the github releases feature.

I can do a bit of research to see if that's possible.

Not sure, but is it preferable to have this built into the actual CI workflow, so that if there comes a time when we want to publish these packages, the versioning would be done as part of the publish? But maybe it's too early to conjecture about how that would actually work anyway.

Reply:
I guess there is the standard web hook mechanism, but we would have to build an endpoint to process that I presume?

https://developer.github.com/v3/activity/events/types/#releaseevent